Broken build

**Describption**
The build has been broken since last changes.

**Expected behavior**
The library should build as usual.

**Sample code**
Part of the log:
```
[7/9] Building CXX object CMakeFiles/amqpcpp.dir/src/array.cpp.o
ninja: job failed: /usr/bin/c++  -isystem /builds/cpp/3rdparty/amqp-cpp/include -O3 -DNDEBUG -std=gnu++11 -MD -MT CMakeFiles/amqpcpp.dir/src/linux_tcp/sslerrorprinter.cpp.o -MF CMakeFiles/amqpcpp.dir/src/linux_tcp/sslerrorprinter.cpp.o.d -o CMakeFiles/amqpcpp.dir/src/linux_tcp/sslerrorprinter.cpp.o -c /builds/cpp/3rdparty/amqp-cpp/src/linux_tcp/sslerrorprinter.cpp
In file included from /builds/cpp/3rdparty/amqp-cpp/src/linux_tcp/sslerrorprinter.cpp:12:
/builds/cpp/3rdparty/amqp-cpp/src/linux_tcp/sslerrorprinter.h:57:17: error: field '_message' has incomplete type 'std::string' {aka 'std::__cxx11::basic_string<char>'}
   57 |     std::string _message;
      |                 ^~~~~~~~
In file included from /usr/include/c++/10.3.1/iosfwd:39,
                 from /usr/include/c++/10.3.1/memory:74,
                 from /builds/cpp/3rdparty/amqp-cpp/src/linux_tcp/sslerrorprinter.h:21,
                 from /builds/cpp/3rdparty/amqp-cpp/src/linux_tcp/sslerrorprinter.cpp:12:
/usr/include/c++/10.3.1/bits/stringfwd.h:74:11: note: declaration of 'std::string' {aka 'class std::__cxx11::basic_string<char>'}
   74 |     class basic_string;
      |           ^~~~~~~~~~~~
ninja: subcommand failed
```

--- a/src/linux_tcp/sslerrorprinter.h
+++ b/src/linux_tcp/sslerrorprinter.h
@@ -18,6 +18,7 @@
  *  Dependencies
  */
 #include "openssl.h"
+#include <string>
 #include <memory>
 
 /**

